packages/access-client/src/capabilities/store.js

```diff
@@ -130,6 +130,18 @@ export const list = base.derive({
      * be stored.
      */
     with: URI.match({ protocol: 'did:' }),
+    nb: {
+      /**
+       * Item where a previous list operation stopped, inclusive of the previous
```

Comment:
This sounds specific to the particular implementation backing this. Maybe this could be more generic?